manifests/init.pp

```diff
@@ -97,14 +107,14 @@
       owner   => 'root',
       group   => 'root',
       require => Package[$tacplus_pkg],
-      before  => Service['tac_plus'],
+      before  => Service[$tac_plus_service],
     }
   }
 
-  service { 'tac_plus':
+  service { $tac_plus_service:
```

Comment:
Please use a generic name like ```tacas_plus_service``` and use the ```name``` attribute of the service resource to specify the actual service name. With this it you can reference the generic resource name in later dependencies. Makes it easier to read and code.